### PR TITLE
Update Hopac.fsx

### PR DESCRIPTION
**Pull request by haf**
Suggesting

 - how to install the packages without compiling Hopac
 - making the script self-contained, downloading all its own dependencies
 - correcting '.Extra' -> '.Extras`

### CODE REVIEW

File: Hopac.fsx [hunk 1]
```diff
@@ -1,15 +1,40 @@
...
 
-#r "Libs/Hopac.Core/bin/Release/Hopac.Core.dll" ;;
-#r "Libs/Hopac/bin/Release/Hopac.dll" ;;
-#r "Libs/Hopac.Extra/bin/Release/Hopac.Extra.dll" ;;
-#r "Libs/Hopac.Experimental/bin/Release/Hopac.Experimental.dll" ;;
-#r "Libs/Hopac.Platform.Net/bin/Release/Hopac.Platform.dll" ;;
+Environment.CurrentDirectory <- __SOURCE_DIRECTORY__
+
+if not (File.Exists "paket.exe") then
+  let url = "https://github.com/fsprojects/Paket/releases/download/1.5.0/paket.exe"
```
Comment (vasily-kirichenko) [inline Hopac.fsx]: Isn't this link to 1.5.0 stable?

Comment (haf) [inline Hopac.fsx]: Yes

**Comment by VesaKarvonen** [comment]
Hmm... The intention behind Hopac.fsx is somewhat different from what I think you are after here.  I've used Hopac.fsx for some interactive experimentation and I specifically want to use freshly compiled DLLs from the project tree.  The Hopac.fsx script works for me as such when used from Windows and via fsi.exe/fsianycpu.exe.  The fsharpi tool, of course, doesn't quite work the same when it comes to DLL search paths.  Also, there is a Hopac.Extra library (which is on its way out though) so the reference to it is not a bug.

Perhaps both the original Hopac.fsx and the modified version should be included with different names?

**Comment by haf** [comment]
@VesaKarvonen Oh I see =). Well yes, splitting them would probably be a good idea -- I followed the getting-started docs which mention it.

**Pull request closed without merging**